File: rabbitmq-jpa-kotlin/core/src/main/code/sample/core/transfer/EmptyResponseWrapper.java
package sample.core.transfer;

import lombok.Data;

@Data
public class EmptyResponseWrapper {

    public static EmptyResponseWrapper create() {
        return new EmptyResponseWrapper();
    }

    private boolean success = true;

    public EmptyResponseWrapper success(boolean success) {
        this.success = success;
        return this;
    }

    private String lang;

    public EmptyResponseWrapper lang(String lang) {
        this.lang = lang;
        return this;
    }

    private String code;

    public EmptyResponseWrapper code(String code) {
        this.code = code;
        return this;
    }

    private String message;

    public EmptyResponseWrapper message(String message) {
        this.message = message;
        return this;
    }
}

File: 2021-rabbitmq-jpa/core/src/main/java/sample/core/transfer/EmptyResponseWrapper.kt
package sample.core.transfer

class EmptyResponseWrapper {
    private var success = true
    fun success(success: Boolean): EmptyResponseWrapper {
        this.success = success
        return this
    }

    private var lang: String? = null
    fun lang(lang: String?): EmptyResponseWrapper {
        this.lang = lang
        return this
    }

    private var code: String? = null
    fun code(code: String?): EmptyResponseWrapper {
        this.code = code
        return this
    }

    private var message: String? = null
    fun message(message: String?): EmptyResponseWrapper {
        this.message = message
        return this
    }

    companion object {
        fun create(): EmptyResponseWrapper {
            return EmptyResponseWrapper()
        }
    }
}
